wasp/apps/clock.py

```diff
@@ -30,6 +30,10 @@ class ClockApp():
     """
     NAME = 'Clock'
     ICON = icons.clock
+    _colon_color = 0xb5b6
+    _left_color = 0xbdb6
+    _right_color = 0xffff
+    _read_theme = False
```

Comment:
Caching theme colours in class variables is a waste of RAM (there is a trade off but RAM is usually more precious than CPU time). These should be looked up in the draw method.

Reply:
sure, should I change it for the widgets as well?

Reply:
changed for widgets too.

Reply:
only widget that was left caching is the Slider widget, and that's because it was already doing caching by itself.

It also does some computations with the color that I don't want to mess with, so it's the only widget atm that's still caching.